### DISCUSSION TITLE
VS code minimum version required

### DISCUSSION THREAD

**Discussion by asifiqbal** [community/community#7104] [Copilot Conversations] [answered]
What is the minimum version of VScode needed to work with github copilot? I get **not compatible** when trying to install the extension with vscode 1.54.3.

**Comment by davecheney** [comment, marked as answer]
VSCode 1.57 or later. VSCodium or OSS Code are not currently supported.

**Reply by Krzysztof-Cieslak** [reply]
However, you can get it working in Code-OSS build using the workaround described in https://github.com/github/feedback/discussions/6629#discussioncomment-1524627

**Comment by asifiqbal** [comment]
Not sure what Code-OSS build is. I have VSCode 1.54.3 on Windows Server
2019 and I added "enable-proposed-api":["github.copilot"] as suggested in #6629
(comment)
<https://github.com/github/feedback/discussions/6629#discussioncomment-1524627>
and
restarted. That did not seem to help. Gitlab Copilot extension is still
failing to install due to the lower version of VScode.

Thanks,
Asif


On Mon, Nov 1, 2021 at 3:17 PM Krzysztof Cieślak ***@***.***>
wrote:

> However, you can get it working in Code-OSS build using the workaround
> described in #6629 (comment)
> <https://github.com/github/feedback/discussions/6629#discussioncomment-1524627>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/github/feedback/discussions/7104#discussioncomment-1572110>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABYKJC7Z4OZWJOG7IV4B53UJ3RV5ANCNFSM5HEOWIWA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Asif Iqbal
PGP Key: 0xE62693C5 KeyServer: pgp.mit.edu
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?

**Reply by Krzysztof-Cieslak** [reply]
Yes, in all cases it requires `1.57`+. My comment was related to `VSCodium or OSS Code are not currently supported.` part of the answer.

**Reply by davecheney** [reply]
@asifiqbal please upgrade to the latest VScode, versions older than 1.57 are not supported.

**Comment by slackingfred** [comment]
Hi there, now it's 2024, so the minimum requirement must has changed, right?
I just received a deprecation notice on VS Code 1.79.2...
